### ISSUE TITLE
Mortal Wounds not Included in Wounds Step

### ISSUE THREAD

**Issue by bschulzke** [toadchild/40kdice#22]
Looks like Mortal Wounds aren't getting included in the expected wounds and the graph for the wounds step. This has a couple weird consequences, like having 0 expected wounds but also expecting 5.19 damage, as we see in this screenshot rolling a bunch of genestealer attacks against a landraider. I'm assuming this is because there are 0 _normal_ wounds, since the genestealers only wound the landraider on 6s, and those are all mortal wounds. Presumably, we want to add mortal wounds and regular wounds together for the calculation of expected wounds, though we'll obviously still want them separate in saving throw step. 

![image](https://github.com/user-attachments/assets/5dd16ba6-e87a-4c06-9ebc-754d84a6d1a3)

**Comment by toadchild** [comment]
You're right, the "Expected" line below the graph doesn't reflect the mortal wounds that are in the graph. I will take a note to add a second note that includes expected mortals.

**Comment by bschulzke** [comment]
> You're right, the "Expected" line below the graph doesn't reflect the mortal wounds that are in the graph. I will take a note to add a second note that includes expected mortals.

Ah you're right, it's just the "Expected line"—I hadn't looked closely at the graph; seems like a reasonable solution.
